server/routes/users.js

```diff
@@ -4,21 +4,23 @@ import express from 'express';
 import fs from 'fs';
 import jdenticon from 'jdenticon';
 import path from 'path';
+import hexToHsl from 'hex-to-hsl';
 import { AVATAR_CONFIG, JDENTICON_CONFIG } from '../constants/config';
 
-var router = express.Router();
-jdenticon.config = JDENTICON_CONFIG;
+let router = express.Router();
 
 /* GET user avatar - jdenticon or avataar . */
 router.get('/:address?', function (req, res, next) {
   const address = req.params.address || '';
-  const fileName = path.resolve(__dirname, `../../public/avatars/${address || 'anonymous'}`);
+  const color = req.query.color
 
+  const fileName = path.resolve(__dirname, `../../public/avatars/${address || 'anonymous'}`);
+  const config = { ...JDENTICON_CONFIG, ...{ hues: color ? [hexToHsl(color)[0]] : [] } }
```

Comment:
```suggestion
  const config = { ...JDENTICON_CONFIG, hues: color ? [hexToHsl(color)[0]] : [] }
```
why do accept the whole color if only hue component needed?